build-system/tasks/prepend-global/index.js

```diff
@@ -136,26 +137,42 @@ function applyConfig(
   return checkoutBranchConfigs(filename, opt_localBranch, opt_branch)
     .then(() => {
       return Promise.all([
-        fs.promises.readFile(filename),
-        fs.promises.readFile(target),
+        fs.promises.readFile(filename, 'utf8'),
+        fs.promises.readFile(target, 'utf8'),
+        fs.promises.readFile(customConfigFile, 'utf8').catch(() => {}),
       ]);
     })
     .then((files) => {
+      let configString = files[0];
+      const targetString = files[1];
+      const overlayString = files[2];
+
       let configJson;
       try {
-        configJson = JSON.parse(files[0].toString());
```

Comment:
Was the old `toString()` here superfluous?

Reply:
When you specify an encoding option (notice `utf8` was added to each `readFile()`), then string content is automatically returned instead of `Buffer`. [ref](https://nodejs.org/docs/latest-v12.x/api/fs.html#fs_fs_readfile_path_options_callback)